compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -399,9 +399,15 @@ object Flags {
   /** A module variable (Scala 2.x only) */
   final val Scala2ModuleVar = termFlag(57, "<modulevar>")
 
+  /** A Scala 2.12 trait that has been augmented with static members */
+  final val Scala_2_12_Augmented = typeFlag(57, "<scala12augmented")
```

Comment:
I don't know the convention about type flag names, but this is the only one with `<`, but without `>`.
Should it be `<scala12augmented>` instead?